test/testModel.cc

```diff
@@ -430,11 +430,12 @@ SUITE(Canvas)
     {
       cairo::Surface surf(cairo_recording_surface_create(CAIRO_CONTENT_COLOR,nullptr));
       c->draw(surf.cairo());// reposition ports
-      CHECK(c->clickType(c->x(),c->y()) == ClickType::onItem);
-      canvas.mouseDown(c->x(),c->y());
-      canvas.mouseUp(400,500);
-      CHECK_EQUAL(400,c->x());
-      CHECK_EQUAL(500,c->y());
+      RenderVariable rv(*dynamic_cast<VariableBase*>(c.get()));
+      CHECK(c->clickType(c->x()+0.5*rv.width(),c->y()+0.5*rv.height()) == ClickType::onItem); // offset added because resize handles grabbed otherwise, for feature 94
+      canvas.mouseDown(c->x()+0.5*rv.width(),c->y()+0.5*rv.height());
+      canvas.mouseUp(500,500);
+      CHECK_EQUAL(500,c->x()+0.5*rv.width());
+      CHECK_EQUAL(500,c->y()+0.5*rv.height());
```

Comment:
I think you've changed the nature of this test. The former test tests that the item is moved to (400,500). The latter test tests that the item was resized such that the top right corner is at (500,500).

Both are valid tests to have in the regression suite,

Reply:
OTOH, clickType is returning onItem. Should be onResizeHandle...

Reply:
This is proving to be somewhat more difficult than I had expected. The resize handles only appear in the correct place when I first create the default canvas item (a note). After grabbing the handles and pulling them, the resize handles then only reappear depending on the extent of zoom-in/out. The more zoomed in, the more easily I can "find" them, but they also appear slightly offset from their original position (towards the center of the note's bounding box). Reminds me of the problem Kresimir experiences with the disappearing canvas items...